src/components/Page3.jsx

```
@@ -1,14 +1,78 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)